src/materials_processor/usd/texture_builders/arnold.py

```
@@ -0,0 +1,308 @@
+        shader_usdshade.CreateInput('thin_film_thickness', Sdf.ValueTypeNames.Float).Set(0)
+        shader_usdshade.CreateInput('thin_walled', Sdf.ValueTypeNames.Bool).Set(False)
+        shader_usdshade.CreateInput('transmit_aovs', Sdf.ValueTypeNames.Bool).Set(False)
+
+    def _arnold_initialize_image_shader(self, image_path: str):
+        image_shader = UsdShade.Shader.Define(self.stage, image_path)
+        image_shader.CreateIdAttr("arnold:image")
+
+        color_space = image_shader.CreateInput("color_space", Sdf.ValueTypeNames.String)
+        color_space.Set("auto")
+        file_input = image_shader.CreateInput("filename", Sdf.ValueTypeNames.Asset)
+        filter = image_shader.CreateInput("filter", Sdf.ValueTypeNames.String)
```

Ruff (0.15.14)

[error] Variable `filter` is shadowing a Python builtin

(A001)